worksheet_B/README.md

```diff
@@ -0,0 +1,21 @@
+PSEUDOCODE:
+
+```
+Insert word
+if correct guess
+    Access granted 
+elif
+    subtract attempts remaining by 1       
+else
+    end minigame when no attempts remain       
+print "TERMINAL LOCKED"
+```
+
+
+
+FLOWCHART: ![fo4 flowchart](https://i.imgur.com/v1GjlIc.png)
```

Comment:
Flowchart looks reasonable, however please review the proper usage of flowchart symbols. E.g. instead of the decision (diamond) node labelled "select and guess a word", it would be more conventional to have a process (rectangle) node with this label, followed by a decision node labelled "is the guess correct?" with arrows labelled "yes" and "no". Similarly the check for number of incorrect guesses remaining should be a decision node.